tests/fakeprotocol_smartcam.py

```diff
@@ -221,35 +221,38 @@ async def _send_request(self, request_dict: dict):
                 return {**result, "error_code": 0}
             else:
                 return {"error_code": -1}
-        elif method[:3] == "get":
+
+        if method in info:
             params = request_dict.get("params")
-            if method in info:
-                result = copy.deepcopy(info[method])
```

Comment:
With `scanApList` added this needs to check for the method in `info` regardless of whether it starts with `get`. No need for extra nesting also.